src/utils.ts

```diff
@@ -23,7 +23,7 @@ export function prefixStorage (storage: Storage, base: string): Storage {
   const nsStorage: Storage = { ...storage }
   for (const prop of storageKeyProps) {
     // @ts-ignore Better types?
-    nsStorage[prop] = (key: string, ...args) => storage[prop](base + key, ...args)
+    nsStorage[prop] = (key: string = '', ...args) => storage[prop](normalizeKey(base + key), ...args)
```

Comment:
`normalizeBase` ensures we have `:` in the end of the base and storage methods always normalize the final key. Do you have an example of why this doesn't works? (adding to fixture would be bonous!)

Reply:
> storage methods always normalize the final key.

You are right, `normalizeKey` is redundant here

About the default value for key:
When we call `storage.getKeys()`, the `key` will be `undefined` and therefor `base + key` will become `BASE:undefined` which is wrong behavior